doc/Command-Reference.md

```diff
@@ -8766,6 +8815,43 @@ This command is used to delete the configured syslog server.
   Running command: systemctl restart rsyslog-config
   ```
 
+**config syslog rate-limit-host**
+
+This command is used to configure syslog rate limit for host.
+
+- Usage:
+  ```
+  config syslog rate-limit-host
+  ```
+
+- Parameters:
+  - _interval_: determines the amount of time that is being measured for rate limiting.
+  - _burst_: defines the amount of messages, that have to occur in the time limit of interval, to trigger rate limiting
+
+- Example:
+  ```
+  admin@sonic:~$ sudo config syslog rate-limit-host --interval 300 --burst 20000
+  ```
+
+**config syslog rate-limit-container**
+
+This command is used to configure syslog rate limit for containers.
+
+- Usage:
+  ```
+  config syslog rate-limit-container
+  ```
+
+- Parameters:
+  - _interval_: determines the amount of time that is being measured for rate limiting.
+  - _burst_: defines the amount of messages, that have to occur in the time limit of interval, to trigger rate limiting
+
+- Example:
+  ```
+  admin@sonic:~$ sudo config syslog rate-limit-container bgp --interval 300 --burst 20000
+  ```
```

Comment:
if there is a default value, suggest to add as well
if there is range supported, suggest to add as well

Reply:
There is no default value, either interval or burst must be present. There is no range supported, as we did not find range from rsyslog document.